test-utils/fixtures/BaseFixtures.scala

```diff
@@ -54,6 +54,7 @@ trait BaseFixtures {
   val testSubmissionDate: LocalDateTime = LocalDateTime.now()
   val testConfirmationReference: String = "UYVQBLMXCYK6HAEBZI7TSWAQ6XDTXFYU"
   val testOnwardRoute: Call = Call("GET", "/foo")
+  val testOnwardRouteTwo: Call = Call("GET", "/bar")
```

Comment:
I don't think this is needed universally, so can be defined wherever it's needed